config/plugins.js

```diff
@@ -1,10 +1,14 @@
 const postCssCustomMedia = require("postcss-custom-media")
+const postCssUrl = require("postcss-url")
 
 module.exports = root => [
   {
     resolve: "gatsby-plugin-sass",
     options: {
-      postCssPlugins: [postCssCustomMedia()],
+      postCssPlugins: [
+        postCssCustomMedia(),
+        postCssUrl([{ filter: "**/fonts/inline/*", url: "inline" }]),
```

Comment:
Does this inline stuff??????????????!!!!!!!!! :scream: It inlines the whole font still, I believe

Reply:
According to the plugin page that's what that does. :stuck_out_tongue: 

Hard to check in development, but testing it in the deploy preview https://deploy-preview-290--subvisual.netlify.com/, disabling cache there is no FOUT in the title. So seems like its working.

Reply:
No javascript fall-back is screwed though, on it.

Reply:
Fixed

Reply:
44k just the HTML doc, seems fine.

Reply:
Just for info purposes:
- New _Acta Display_ limited to the subset: 9.4KB WOFF + 7.4KB WOFF2 (this is what we inline);
- _Acta Display_ without tittle with only the `i`: 3.0KB WOFF + 2.2KB WOFF2 (this we load async).

We can inline only one of them to keep things even smaller. According to https://caniuse.com/#search=woff WOFF2 is only missing support in IE, but that would cost us only 2KB. What say you?